docs/tables/aws_appconfig_application.md

```diff
@@ -0,0 +1,18 @@
+# Table: aws_appconfig_application
+
+In AWS AppConfig, an application is simply an organizational construct like a folder. This organizational construct has a relationship with some unit of executable code.
+
+## Examples
+
+### Basic info
```

Comment:
Based off of the lack of examples and useful columns, it's hard to tell why this table is required. @karanpopat @khushboo9024 Do you know why the request came up to create this table? Is it meant to join with another table's data? Do we need other AppConfig tables for this table to be useful?

Reply:
While this table by itself is not very useful, later we will add tables like `aws_appconfig_environment`, which will use this table as a parent hydrate. So we'll add this table for now and then add additional AppConfig tables later on.